HelpMyStreetFE/HelpMyStreetFE/src/js/requesthelp/request-stage.js

```diff
@@ -136,6 +136,35 @@ var handleActivity = function (el) {
     let taskId = el.attr("data-id");
     $('input[name="currentStep.SelectedTask.Id"]').val(taskId);
     LoadQuestions(taskId);
+
+
+    
+    if (taskId == 2) { // facemask   
+        $('#requestorFor_3').show(); // onbehalf of organisation
+        if ($('#requestorFor_3').hasClass("selected")) {
+            $('input[name="currentStep.SelectedRequestor.Id"]').val($('#requestorFor_3').attr("data-id"));
```

Comment:
This if TaskID ==2 code isnt new... i just moved out of the GetQuestionsFunction Callback (had no reason to be in there)